packages/inventories-exporter/src/cli.js

```diff
@@ -0,0 +1,159 @@
+import fs from 'fs'
+import { getCredentials } from '@commercetools/get-credentials'
+import npmlog from 'npmlog'
+import PrettyError from 'pretty-error'
+import yargs from 'yargs'
+
+import CONSTANTS from './constants'
+import InventoryExporter from './main'
+import { description } from '../package.json'
+
+process.title = 'inventories-exporter'
+
+const args = yargs
+  .usage(
+    `\n
+Usage: $0 [options]
+${description}`,
+  )
+  .showHelpOnFail(false)
+
+  .option('help', {
+    alias: 'h',
+  })
+  .help('help', 'Show help text.')
+
+  .version()
+
+  .option('outputFile', {
```

Comment:
Nitpick: I think you can use `output` here instead of `outputFile`. Looks better if you're piping to a non-file stream